druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -755,28 +779,16 @@ private void checkGroupBySingleSortLimit(boolean approx) {
   /** Tests a query that contains no GROUP BY and is therefore executed as a
    * Druid "select" query. */
   @Test public void testFilterSortDesc() {
-    final String sql = "select * from \"foodmart\"\n"
```

Comment:
I would say that failing for ```select *...``` is not an option, since user just wants "all visible columns".

Thus, could the expansion logic check the table metadata to skip virtual columns?